src/interfaces/init.h

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2019 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_INTERFACES_INIT_H
+#define BITCOIN_INTERFACES_INIT_H
+
+#include <fs.h>
+#include <functional>
+#include <memory>
+#include <string>
+#include <vector>
+
+struct NodeContext;
+
+namespace interfaces {
+class Base;
+class IpcProcess;
+class IpcProtocol;
+
+//! Init interface providing access to other interfaces. The Init interface is
```

Comment:
The documentation you wrote down in review club notes ("The Init interface is similar to other cross-process C++ interfaces like interfaces::Node, interfaces::Wallet, interfaces::Chain and interfaces::ChainClient, providing virtual methods that can be called from other processes....") is describing well design goal of `Init` IMO. Maybe just pick it up ?